libs/auth-api-lib/migrations/20221006143438-add-domain-fields.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface) {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+
+        ALTER TABLE domain
+          ADD COLUMN display_name VARCHAR DEFAULT 'Mínar síður Ísland.is' NOT NULL;
+
+        ALTER TABLE domain
+          ADD COLUMN organisation_logo_key VARCHAR DEFAULT 'Stafrænt Ísland' NOT NULL;
+          
+      COMMIT;
+    `)
+  },
+
+  async down(queryInterface) {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+
+        ALTER TABLE domain
+          DROP COLUMN display_name;
+
+        ALTER TABLE domain
+          DROP COLUMN organisation_logo_key;
+
+      COMMIT;
+    `)
+  },
```

Comment:
We should prefer sequelize's `addColumn` and `removeColumn` over raw sql queries.